packages/frontend/src/pages/campaigns/top-nav/index.tsx

```diff
@@ -26,8 +33,21 @@ export const CampaignsTopNav = ({
     onStateChange,
     onToggleFilters,
     filtersOpen,
-}: // onSearchQueryChange,
-CampaignsTopNavProps) => {
+    setSearchQuery,
+}: CampaignsTopNavProps) => {
+    const [searchInputValue, setSearchInputValue] = useState<
+        string | undefined
+    >();
+
+    const handleSearchChange = useCallback(
+        (event: ChangeEvent<HTMLInputElement>): void => {
+            const inputValue = event.currentTarget.value;
```

Comment:
Unsure why `currentTarget` instead of `target`? I've always used `target` and I'm not sure about the differences.

Reply:
I had the types wrong before. It's working correctly. Fixed. Thanks.